### PR TITLE
Improve readability of product tests execution logs

### PR DESCRIPTION
**Pull request by wendigo**
This PR modifies how different information is displayed while running product tests. Here are the examples:

* List of containers that are created during the environment startup:

```
+---------------+-------------------+---------------------------+---------+----------------------------------------------------------------------------+
|     container |              name |                     image | startup |                                                                      ports |
+---------------+-------------------+---------------------------+---------+----------------------------------------------------------------------------+
|         tests |         ptl-tests |  prestodev/centos6-oj8:34 |  7.77ms |                                                                            |
| presto-master | ptl-presto-master | prestodev/centos7-oj11:34 |  44.02s |                                                                       8080 |
| hadoop-master | ptl-hadoop-master |  prestodev/hdp2.6-hive:34 |  22.89s | 1180, 8020, 8042, 8088, 9000, 9083, 9864, 9870, 10000, 19888, 50070, 50075 |
+---------------+-------------------+---------------------------+---------+----------------------------------------------------------------------------+
```

* List of test runs in a suite:

```
+-------------------------------------------------------------+--------------------+-----------------+----------------------------------------------------+----------------+
|                                                 environment |             groups | excluded groups |                                              tests | excluded tests |
+-------------------------------------------------------------+--------------------+-----------------+----------------------------------------------------+----------------+
|                                               multinode-tls |              smoke |     skip_on_cdh |                                                    |                |
|                                                             |                cli |         iceberg |                                                    |                |
|                                                             |           group-by |                 |                                                    |                |
|                                                             |               join |                 |                                                    |                |
|                                                             |                tls |                 |                                                    |                |
+-------------------------------------------------------------+--------------------+-----------------+----------------------------------------------------+----------------+
|                                      multinode-tls-kerberos |                cli |     skip_on_cdh |                                                    |                |
|                                                             |           group-by |         iceberg |                                                    |                |
|                                                             |               join |                 |                                                    |                |
|                                                             |                tls |                 |                                                    |                |
+-------------------------------------------------------------+--------------------+-----------------+----------------------------------------------------+----------------+
| singlenode-kerberos-hdfs-impersonation-with-wire-encryption |    storage_formats |     skip_on_cdh |                                                    |                |
|                                                             |                cli |         iceberg |                                                    |                |
|                                                             | hdfs_impersonation |                 |                                                    |                |
|                                                             |      authorization |                 |                                                    |                |
+-------------------------------------------------------------+--------------------+-----------------+----------------------------------------------------+----------------+
| singlenode-kerberos-hdfs-impersonation-with-data-protection |                    |     skip_on_cdh | TestHiveStorageFormats.testOrcTableCreatedInPresto |                |
|                                                             |                    |         iceberg |                TestHiveCreateTable.testCreateTable |                |
+-------------------------------------------------------------+--------------------+-----------------+----------------------------------------------------+----------------+
```

* Suite results:

```
+----------------------------------+------------+-------------+----------------+---------+---------+--------------------------+
|                               id |      suite | environment |         config |  status | elapsed |                    error |
+----------------------------------+------------+-------------+----------------+---------+---------+--------------------------+
| 86c31ba456914714acb8d4b3f795876e | suite-test |  singlenode | config-default | SUCCESS |   1.39m |                        - |
| 65254e18fad84b84bb68d16847b1119e | suite-test |   multinode | config-default |  FAILED |   1.60m | Tests exited with code 1 |
+----------------------------------+------------+-------------+----------------+---------+---------+--------------------------+
```

The `suite run id` was changed from a numeric index to a random UUID. I've added a marker in the suite execution that allows to quickly find where the suite started and finished.

### CODE REVIEW

File: presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/util/ConsoleTable.java [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+import java.util.stream.Collectors;
+
+import static io.prestosql.tests.product.launcher.util.ConsoleTable.Alignment.RIGHT;
+import static java.util.Objects.requireNonNull;
+
+public class ConsoleTable
+{
+    private ImmutableList.Builder<TableElement> builder = ImmutableList.builder();
+
+    public void addLine()
```
Comment (losipiuk) [inline presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/util/ConsoleTable.java]: name it `AddSeparator`

**Review by losipiuk — APPROVED** [review]
skimmed

**Comment by findepi** [comment]
The more complexity we add, the more we need some tests for the Launcher. And @findepi started that project with no testability in mind :/

**Comment by wendigo** [comment]
> The more complexity we add, the more we need some tests for the Launcher. And @findepi started that project with no testability in mind :/ 

I'm thinking about adding those

**Comment by losipiuk** [comment]
> The more complexity we add, the more we need some tests for the Launcher. And @findepi started that project with no testability in mind :/

Agreed. Though I see this specific change very much outside of the core of what launcher does. A nice pet weekend project.

I do not think we should invest time in adding test coverage for launcher. Unless we see that lack of those kick us hard often.

**Comment by wendigo** [comment]
@losipiuk what about a stress test (that uses launcher to run single test and single, test suite in a loop (for i.e. 30 minutes) to test for stability etc?

**Comment by losipiuk** [comment]
> @losipiuk what about a stress test (that uses launcher to run single test and single, test suite in a loop (for i.e. 30 minutes) to test for stability etc?

Maybe. Though I am not super convinced that it gives us much more than just using launcher in CI and observing potential issues with actual CI runs.

**Comment by findepi** [comment]
The benefit would be that tests are focused on specific functionalities. For example, we do not know that logigng of container states works, until we actually need it and look for it.

But, i see the point @losipiuk  -- it maybe the investment does not pay off.

**Pull request merged**